### ISSUE TITLE
json export is missing frame numbers

### ISSUE THREAD

**Issue by daisyowl** [aseprite/aseprite#1889]
If i export a sprite animation with options including `--filename-format '{path}/{title}_{innertag}_{tagframe}.{extension}'`, the json file saved alongside the .pngs is formatted like this:

(note that the image paths don't contain the frame number of the animation tagged "groove" in the document "jeffFull")

(also the actual json document has the full path in the key, which i stripped out for clarity, but would also like to be able to suppress in the CLI output)

```
...
   "jeffFull_groove_.ase": {
    "frame": { "x": 1260, "y": 0, "w": 36, "h": 36 },
    "rotated": false,
    "trimmed": false,
    "spriteSourceSize": { "x": 0, "y": 0, "w": 36, "h": 36 },
    "sourceSize": { "w": 36, "h": 36 },
    "duration": 70
   },
   "jeffFull_groove_.ase": {
    "frame": { "x": 1296, "y": 0, "w": 36, "h": 36 },
    "rotated": false,
    "trimmed": false,
    "spriteSourceSize": { "x": 0, "y": 0, "w": 36, "h": 36 },
    "sourceSize": { "w": 36, "h": 36 },
    "duration": 70
   },
   "jeffFull_groove_.ase": {
    "frame": { "x": 1332, "y": 0, "w": 36, "h": 36 },
    "rotated": false,
    "trimmed": false,
    "spriteSourceSize": { "x": 0, "y": 0, "w": 36, "h": 36 },
    "sourceSize": { "w": 36, "h": 36 },
    "duration": 70
   },
   "jeffFull_groove_.ase": {
    "frame": { "x": 1368, "y": 0, "w": 36, "h": 36 },
    "rotated": false,
    "trimmed": false,
    "spriteSourceSize": { "x": 0, "y": 0, "w": 36, "h": 36 },
    "sourceSize": { "w": 36, "h": 36 },
    "duration": 70
   },
   "jeffFull_groove_.ase": {
    "frame": { "x": 1404, "y": 0, "w": 36, "h": 36 },
    "rotated": false,
    "trimmed": false,
    "spriteSourceSize": { "x": 0, "y": 0, "w": 36, "h": 36 },
    "sourceSize": { "w": 36, "h": 36 },
    "duration": 70
   },
   "jeffFull_groove_.ase": {
    "frame": { "x": 1440, "y": 0, "w": 36, "h": 36 },
    "rotated": false,
    "trimmed": false,
    "spriteSourceSize": { "x": 0, "y": 0, "w": 36, "h": 36 },
    "sourceSize": { "w": 36, "h": 36 },
    "duration": 70
   },
...
```

aseprite 1.2.9x64, win10

**Comment by daisyowl** [comment]
i should also note that the rest of the export works fine, the actual numbered images are exported correctly

**Comment by dacap** [comment]
Hi @daisyowl, are all frames inside a tag? or you aren't using tags? did you try using just `{frame}` instead of `{tagframe}`?

**Comment by daisyowl** [comment]
I'm using tags, I'd like to export with `{tagFrame}` instead of `{frame}` because i want the frame numbers to be relative to the tags. Here's an example file:

[jeffFull.zip](https://github.com/aseprite/aseprite/files/2458691/jeffFull.zip)

**Comment by dacap** [comment]
Sorry, definitely it's a bug, I'll try to fix this ASAP for v1.2.10.

**Comment by daisyowl** [comment]
great, thank you!
